spec/wishlist.spec.js

```diff
@@ -0,0 +1,30 @@
+import Wishlist from '../src/api/wishlist';
+
+jest.mock('../src/api/base');
+
+describe('Wishlist should add a product to the wishlist', () => {
+    const mockedUrl = '/wishlist.php?action=add&wishlistid=14&product_id=111';
+    let mokedCallback;
+    let wishlist;
+
+    beforeEach(() => {
+        mokedCallback = jest.fn();
+        wishlist = new Wishlist();
+    });
+
+    test('without options', () => {
```

Comment:
I would recommend to name it as _should add a product to the wishlist without options_. It would be nice to follow pattern of _Should_ExpectedBehavior_When_StateUnderTest_

Reply:
@jairo-bc done!